### ISSUE TITLE
[feature request] search package name

### ISSUE THREAD

**Issue by ghost** [MM2-0/Kvaesitso#7]
As the search is core functionality of the launcher, can it be extended to package name? 
Just like what the "blue line console" do?

**Comment by MM2-0** [comment]
Is that really a thing that people do? I can surely implement it, shouldn't be a difficult thing to do, I just didn't think that this would be a useful feature.

**Comment by J4gQBqqR** [comment]
This is a temporary substitute for #8, since some of (like 50%) the package names are just pinyin or hiragana. However, #8 is the correct way to do it.

**Comment by J4gQBqqR** [comment]
I can imagine this feature will be confusing to some people as they would say, "Why did I enter X and the search returned Y? I did not search for it."

The best practice would be having a toggle to let the user enable/disable it.

**Comment by MM2-0** [comment]
Yeah, could be confusing if you type in "com" or any other TLD and suddently half of your apps are still there. We could limit it to 4-letter+ search queries to avoid that though.
